gm4_fulcio_shamir/data/gm4_fulcio_shamir/functions/create_block.mcfunction

```diff
@@ -0,0 +1,5 @@
+#run from active
+#@s = players holding a fulcio compass and looking at air positioned ^ ^ ^3 align xyz
+
+setblock ~ ~ ~ structure_void keep
+summon area_effect_cloud ~0.5 ~0.5 ~0.5 {Radius:0,Age:-2147483648,CustomName:'"gm4_fulcio_block"',Tags:["gm4_fulcio_block"],Particle:"composter"}
```

Comment:
How about ~6.8 years:

`Age:-2147483648,Duration:-1,WaitTime:-2147483648`

Reply:
Doesn't matter really, this technically should've been set to 17 cuz all AECs with this tag are killed every cycle, but I just wanted some extra time in case for some reason it wasn't killed when it should have died, thus leaving a random structure void in the world